### DISCUSSION TITLE
JPEG-Image and Distance-Image together

### DISCUSSION THREAD

**Discussion by j-friedl** [ifm/ifm3d#363] [Q&A] [answered]
Do I understand correctly, that it is currently not possible to retrieve a JPEG-Image and a Distance-Image at the same time? Is it planned to add this feature? It would be really nice to have for measuring the distance to objects.

**Comment by graugans** [comment, marked as answer]
Hej @j-friedl it is possible, but not through one connection. What you can do is to open two ports and retrieve the 2D data on one port and the 3D data on another port. Currently they are not synchronized. But with a later firmware you will be able to synchronize the cameras. Which will allow you to have multiple cameras running the same framerate and they will be triggered at the same time.

**Reply by j-friedl** [reply]
@graugans sounds great, thank you!
